### DISCUSSION TITLE
Postman 사용 여부

### DISCUSSION THREAD

**Discussion by nomatterjun** [Favor-Gift-Reminder/Favor-iOS#71] [Ideas]
로그인 구현하면서 생각을 해본건데

사실 서버 배포를 보채는 것보다는 포스트맨 Mock Server를 사용할 수도 있겠다 싶어서..!

명세서 제공해주는거대로 만들거나 포스트맨에 같이 들어가서 직접 수정을 해줘도 되고
괜찮지 않을까..?

**Comment by EnnzoKim** [comment]
`Mock Server` 라는게 정확히 어떤걸까?
나는 요청을 보내고 응답 받는 것 밖에 안 해봐서 ㅠㅠ ..

**Comment by nomatterjun** [comment]
말그대로 일종의 가짜 서버야...!

![스크린샷 2023-02-26 21 23 24](https://user-images.githubusercontent.com/60438045/221410249-ad54ba07-59c7-4258-aeba-0e85d01b18e4.png)

포스트맨에서 제공해주는 baseURL을 통해서 API 콜을 시뮬레이션 할 수 있어
요청이랑 응답을 정해주면 그거대로 작동하는 방식이라

![스크린샷 2023-02-26 21 36 38](https://user-images.githubusercontent.com/60438045/221410858-9e7ea9fe-cb9c-45d8-bf7f-0c251c1f970d.png)

예를 들어서 위 같은 요청을 만들고

![스크린샷 2023-02-26 21 37 01](https://user-images.githubusercontent.com/60438045/221410881-a0cab77f-c360-414c-a8dc-ea72c03395d2.png)

이런 응답을 만들면 요청 URL로 요청을 보내면 실제로

![스크린샷 2023-02-26 21 37 35](https://user-images.githubusercontent.com/60438045/221410918-4b0d56e6-cd8c-4669-93fe-8380b9f98201.png)

이런 응답을 보내줘!

![스크린샷 2023-02-26 21 38 56](https://user-images.githubusercontent.com/60438045/221410982-20c7c85b-8f25-46de-a628-810c3f17514b.png)
![스크린샷 2023-02-26 21 39 05](https://user-images.githubusercontent.com/60438045/221410985-02c6c25a-c904-4d5e-a25d-0c064f1baa07.png)

이렇게..!

그래서 실제로 앱에 네트워킹 기능을 넣어줄 수 있고
실제 서버랑 동일하게 만들어두면 이후에 서버가 배포된 후에는 baseURL만 바꿔주면 되기 떄문에
서버 입장에서도 부담이 좀 덜어지는 효과가 있을 것 같아

프론트 인원들은 동일한 환경에서 테스팅해볼 수 있고..!

**Reply by EnnzoKim** [reply]
오 좋은 것 같아 ! 
이걸 사용하기 위해서 내가 할 일은 뭘까 ..?

**Reply by nomatterjun** [reply]
당장은 없을 것 같아..!

초대해주면 들어와서 기능 살펴보기 정도..?

은기가 월~화요일동안 응답 메시지 넣어주겠다고 했어!
